cmake/ignition-config.cmake.in

```diff
@@ -58,9 +58,9 @@ set(@PKG_NAME@_CONFIG_INCLUDED TRUE)
 
 # Get access to the find_dependency utility
 include(CMakeFindDependencyMacro)
-# Find ignition-cmake, because we need its modules in order to find the rest of
+# Find gz-cmake, because we need its modules in order to find the rest of
 # our dependencies.
-find_dependency(ignition-cmake@IGNITION_CMAKE_VERSION_MAJOR@)
+find_dependency(@PROJECT_NAME@)
```

Comment:
this is a mistake

Reply:
fixed in https://github.com/ignitionrobotics/ign-cmake/pull/244/commits/e44d9dc0bfd2c715238247d6465a7447cb8b0089

Reply:
fixed in https://github.com/ignitionrobotics/ign-cmake/pull/244/commits/e44d9dc0bfd2c715238247d6465a7447cb8b0089